src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor_state.go

```diff
@@ -29,14 +26,12 @@ func (st *StoragePowerActorState_I) _slashPledgeCollateral(rt Runtime, address a
 		currEntry.Impl().LockedPledgeCollateral_ = 0
 		// TODO: extra handling of not having enough pledgecollateral to be slashed
 	} else {
-		currEntry.Impl().LockedPledgeCollateral_ = currEntry.LockedPledgeCollateral() - amount
+		currEntry.Impl().LockedPledgeCollateral_ = currEntry.LockedPledgeCollateral() - amountToSlash
 	}
 
-	// TODO: send amountToSlash to TreasuryActor
-	panic(amountToSlash)
+	st.Impl().PledgeCollateralSlashed_ += amountToSlash
```

Comment:
So we're not using the burnt funds actor? Will the storage market actor similarly hold on to slashed deal collateral? If so we should delete reference to the burnt funds actor from the spec entirely.

Reply:
I agree this should go to the burnt funds actor (which, I believe, may get renamed to the treasury actor).

Reply:
we will still use that, will add that either in this PR or a future one.